service/src/main/java/greencity/service/ubs/SuperAdminServiceImpl.java

```diff
@@ -595,6 +596,14 @@ private Courier tryToFindCourier(Long courierId) {
             .orElseThrow(() -> new EntityNotFoundException(ErrorMessage.COURIER_IS_NOT_FOUND_BY_ID + courierId));
     }
 
+    @Override
+    public boolean checkIfTariffExists(AddNewTariffDto addNewTariffDto) {
+        List<TariffLocation> tariffLocations = tariffsLocationRepository.findAllByCourierIdAndLocationIds(
+            addNewTariffDto.getCourierId(), addNewTariffDto.getLocationIdList());
+
+        return (tariffLocations != null && !tariffLocations.isEmpty());
```

Comment:
Try to use null-safe CollectionUtils.isEmpty(), which checks if list is null or empty